src/components/widgets/ElevationWidget.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import {XYPlotTemplate} from 'components/widgets/shared/XYPlotTemplate';
+import {IDefaultWidgetProps} from 'components/widgets/shared/IDefaultWidgetProps';
+import {IPoint} from 'types/IPoint';
+import {BasicHorizontalAxis, BasicVerticalAxis} from 'components/widgets/shared/BasicAxes';
+import {LinearAreaSeries} from 'components/widgets/shared/BasicAreaSeries';
+
+function timeCompare(a: { x: any; }, b: { x: any; }){
+    let timeA = a.x;
+    let timeB = b.x;
+    let comparison = 0;
+
+    if (timeA > timeB) {
+        comparison = 1;
+    } else if (timeA < timeB) {
+        comparison = -1;
+    }
+    return comparison;
+}
+
+export interface IElevationWidgetProps extends IDefaultWidgetProps {
+  elevation: IPoint[];
+
+  heightPx: number;
+  widthPx: number;
+}
+
+export const ElevationWidget: React.FC<IElevationWidgetProps> = props => (
+  <XYPlotTemplate
+    heightPx={props.heightPx}
+    widthPx={props.widthPx}
+    status={props.elevation.length > 2 && props.elevation.length > 2 ? 'ready' : 'loading'}
+    title="Elevation over Predicated Time"
+    useHorizontalGridLines={true}>
+    {BasicHorizontalAxis({
+      axisLabel: "Time (Minutes)",
+    })}
+
+    {BasicVerticalAxis({
+      axisLabel: "Elevation Level"
+    })}
+    
+    {LinearAreaSeries({
+      data: props.elevation.sort(timeCompare),
```

Comment:
The data should be sorted by X by the time it gets here. This is probably a result of using mock data

Reply:
Or will the live source of data for this graph be different from the other graphs?

Reply:
Jonas ways saying that there is a high chance that the live data is not going to be sorted when piped

Reply:
Yep I sort the data in `dataUtil.ts`

Reply:
In this case I'll accept since we are short on time and I'm unsure where the data will be coming from eventually.